index.js

```diff
@@ -59,7 +59,7 @@ async function csrfPlugin (fastify, opts) {
       : this.request.cookies[cookieKey]
     const userInfo = opts ? opts.userInfo : undefined
     if (!secret) {
-      secret = await tokens.secret()
+      secret = await tokens.secretSync()
```

Comment:
should it still be awaited?

Reply:
i also removed the async, i hope thats correct?